packages/build/src/npm-packages/bump.ts

```diff
@@ -13,27 +12,40 @@ export function bumpNpmPackages(
   }
 
   console.info(`mongosh: Bumping package versions to ${version}`);
-  spawnSyncFn(
-    LERNA_BIN,
-    [
-      'version',
-      version,
-      '--no-changelog',
-      '--no-push',
-      '--exact',
-      '--no-git-tag-version',
-      '--force-publish',
-      '--yes',
-    ],
-    {
-      stdio: 'inherit',
-      cwd: PROJECT_ROOT,
-      encoding: 'utf8',
+  const monorepoRootPath = path.resolve(__dirname, '..', '..', '..', '..');
+  const packages = await getPackagesInTopologicalOrder(monorepoRootPath);
+
+  const locations = [monorepoRootPath, ...packages.map((p) => p.location)];
+
+  for (const location of locations) {
+    const packageJsonPath = path.join(location, 'package.json');
+    console.log({ packageJsonPath });
```

Comment:
I'll remove this once CI finishes. Forgot it in there and I don't want to start everything over again :)